### PR TITLE
[MLDBFB-506] add a test case

### PR DESCRIPTION
**Pull request by simlmx**
This was suppose to outline a bug but merging with master just fixed it. @nicolaskruchten you can decide if it's worth merge it anyway.

### CODE REVIEW

File: testing/MLDBFB-506-stats-tbl-sql-expr.py [hunk 1]
```diff
@@ -0,0 +1,104 @@
+mldb = mldb_wrapper.wrap(mldb) # noqa
+
+
+class BowSqlExprTest(MldbUnitTest):
+
+    def check_true(self, q, truth):
```
Comment (nicolaskruchten) [inline testing/MLDBFB-506-stats-tbl-sql-expr.py]: we have a dedicated assert equal query result function... check https://github.com/mldbai/mldb/blob/master/testing/MLDBFB-336-sample_test.py

**Comment by nicolaskruchten** [comment]
+1 when Jenkins passes

**Comment by nicolaskruchten** [comment]
thanks for the test :)

File: testing/MLDBFB-506-stats-tbl-sql-expr.py (hunk 1 above)
Comment (simlmx) [inline testing/MLDBFB-506-stats-tbl-sql-expr.py]: Cool was wondering about that.

**Pull request merged**